web/dashboard/src/components/Sidebar.jsx

```
@@ -0,0 +1,59 @@
+import { useRoute } from 'preact-router';
+
+const navItems = [
+  { href: '/admin/', label: 'Overview', icon: HomeIcon },
+  { href: '/admin/models', label: 'Models', icon: ModelsIcon },
+];
+
+function HomeIcon() {
+  return (
+    <svg class="w-5 h-5" fill="none" stroke="currentColor" viewBox="0 0 24 24">
```

Biome (2.3.14)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)